include/boost/geometry/extensions/triangulation/strategies/cartesian/in_circle_robust.hpp

```diff
@@ -12,7 +12,7 @@
 #ifndef BOOST_GEOMETRY_EXTENSIONS_TRIANGULATION_STRATEGIES_CARTESIAN_IN_CIRCLE_ROBUST_HPP
 #define BOOST_GEOMETRY_EXTENSIONS_TRIANGULATION_STRATEGIES_CARTESIAN_IN_CIRCLE_ROBUST_HPP
 
-#include<boost/geometry/extensions/triangulation/strategies/cartesian/detail/precise_math.hpp>
+#include<boost/geometry/util/precise_math.hpp>
```

Comment:
Or would `formulas` be a better place?

Reply:
Together with `util/series_expansion.hpp` (probably in a different PR).

Reply:
I put it `util` since it contains basic operations (summation, multiplication etc) just like `math.hpp`, formulas are more "geometric" but I am open to move it in a new PR together with other functions if we are convinced by the concept of formulas vs. util.